examples/test_case_management_v1_examples.py

```diff
@@ -0,0 +1,379 @@
+        """
+        remove_watchlist request example
+        """
+        assert case_number is not None
+
+        try:
+            # begin-removeWatchlist
+
+            watchlist_users = [
+                User(realm='IBMid', user_id='abc@ibm.com')
+            ]
+
+            watchlist = case_management_service.remove_watchlist(
+                case_number=case_number,
+                watchlist=watchlist_users,
+            ).get_result()
+
+            print(json.dumps(watchlist, indent=2))
+
+            # end-removeWatchlist
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_add_resource_example(self):
+        """
+        add_resource request example
+        """
+        assert case_number is not None
+        assert resource_crn is not None
+
+        try:
+            # begin-addResource
+
+            resource = case_management_service.add_resource(
+                case_number=case_number,
+                crn=resource_crn,
+                note="This resource is the service that is having the problem.",
+            ).get_result()
+
+            print(json.dumps(resource, indent=2))
+
+            # end-addResource
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_upload_file_example(self):
+        """
+        upload_file request example
+        """
+        assert case_number is not None
+
+        try:
+            # begin-uploadFile
+
+            example_file_content = b'This is the content of the file to upload.'
+
+            file_with_metadata_model = {
+                'data': io.BytesIO(example_file_content).getvalue(),
+                'filename': 'example.log',
+                'content_type': 'application/octet-stream',
+            }
+
+            files_to_upload = [file_with_metadata_model]
+
+            attachment = case_management_service.upload_file(
+                case_number=case_number,
+                file=files_to_upload,
+            ).get_result()
+
+            print(json.dumps(attachment, indent=2))
+
+            # end-uploadFile
+
+            assert attachment is not None
+            assert attachment['id'] is not None
+
+            global attachment_id
+            attachment_id = attachment['id']
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    @needscredentials
+    def test_download_file_example(self):
+        """
+        download_file request example
+        """
+        assert case_number is not None
+        assert attachment_id is not None
+
+        try:
+            # begin-downloadFile
+
+            response = case_management_service.download_file(
+                case_number=case_number,
+                file_id=attachment_id,
+            )
+
+            file = response.get_result()
+
+            print('Attachment content-type:', response.get_headers()['content-type'])
+            print('Attachment contents:', file)
```

Comment:
We'll need to do something slightly different to print out the contents of the downloaded "file".  When I run the examples, i get this output:
```
Attachment content-type: application/octet-stream
Attachment contents: <Response [200]>
```

Reply:
You are totally right. I forgot to check the verbose logs. With the modification I have just pushed it looks like this:
`Attachment contents: b'This is the content of the file to upload.`
WDYT?